apps/docs/src/app/platform/platform-documentation.routes.ts

```diff
@@ -32,57 +32,57 @@ export const ROUTES: Routes = [
+                ],
             },
             {
-
                 path: 'action-bar',
-                component: PlatformActionbarHeaderComponent,
+                component: PlatformActionBarHeaderComponent,
                 children: [
                     { path: '', component: PlatformActionBarDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.actionbar } }]
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.actionbar } },
+                ],
             },
             {
                 path: 'link',
                 component: PlatformLinkHeaderComponent,
                 children: [
                     { path: '', component: PlatformLinkDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.link } }
-                ]
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.link } },
+                ],
             },
             {
                 path: 'menu',
                 component: PlatformMenuHeaderComponent,
                 children: [
                     { path: '', component: PlatformMenuDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.menu } }
-                ]
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.menu } },
+                ],
             },
             {
                 path: 'search-field',
                 component: PlatformSearchFieldHeaderComponent,
                 children: [
                     { path: '', component: PlatformSearchFieldDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.searchField } }
-                ]
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.searchField } },
+                ],
             },
             {
                 path: 'select',
                 component: PlatformSelectHeaderComponent,
                 children: [
                     { path: '', component: PlatformSelectDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.menu } }
-                ]
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.menu } },
+                ],
             },
             {
                 path: 'radio-group',
                 component: PlatformRadioGroupHeaderComponent,
                 children: [
                     { path: '', component: PlatformRadioGroupDocsComponent },
-                    { path: 'api', component: ApiComponent, data: { content: API_FILES.radioGroup } }
-                ]
-            }
-        ]
-    }
+                    { path: 'api', component: ApiComponent, data: { content: API_FILES.radioGroup } },
+                ],
+            },
+        ],
+    },
```

Comment:
Could you add latest prettier changes, that prevent from putting commas at the end of property?